traveller/modules/profile/forms.py

```diff
@@ -0,0 +1,66 @@
+from modules.box__default.auth.models import User
+from init import ModelForm
+from wtforms.fields.html5 import EmailField
+from wtforms.fields import  TextField
+#from wtforms.validators import DataRequired
+from wtforms.validators import Email
+from wtforms.validators import InputRequired
+#from wtforms.validators import Length
+#from wtforms.validators import EqualTo
+from wtforms.validators import ValidationError
+from sqlalchemy import func
+
+
+def validate_email(self, field):
+            user = User.query.filter(
+                func.lower(User.email) == func.lower(field.data)
+            ).scalar()
+            if user is not None:
+                raise ValidationError(f"email '{field.data}' is already in use.")
```

Comment:
what happened there with the indentation?

Reply:
Yes, because unfortunately Python only requires indentation to be consistent within a block, not within the whole file..

Reply:
Oh, sorry, I ommited this.
It seems like pushed Tab and don`t notice...